### PR TITLE
Little change

### PR DESCRIPTION
**Pull request by taufiqsumadi**
I found the difference in the number. @bil-elmoussaoui

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -176,11 +176,11 @@ msgstr "Mot de passe de protection"
 
 #: TwoFactorAuth/widgets/settings.py:93
 msgid "Auto-lock the application (m):"
-msgstr "Auto-verrouillage de l'application (m) :"
+msgstr "Auto-verrouillage de l'application (m):"
 
 #: TwoFactorAuth/widgets/settings.py:112
 msgid "Secret code generation time (s) :"
```
Comment (jayvdb) [inline po/fr.po]: This extra whitespace looks like a problem in the source message.  It is inconsistent with the other messages.

@bil-elmoussaoui, should the source string be fixed to be `"Secret code generation time (s):"` ?

Comment (bilelmoussaoui) [reply po/fr.po]: @jayvdb You're right! the source strings sucks! if anyone of you wants to update it feel free to do so! Thanks

Comment (taufiqsumadi) [reply po/fr.po]: okay already completed to update it @jayvdb @bil-elmoussaoui

**Comment by bilelmoussaoui** [comment]
Thanks for the PR

**Comment by jayvdb** [comment]
We've been told that the extra space before `:` in `msgstr "Auto-verrouillage de l'application (m) :"` was actually correct French.
And the weblate [message](https://hosted.weblate.org/translate/gnome-twofactorauth/translation/fr/?checksum=03163aa751be1ccf911a216c9c0087cb#history) cant be "fixed" as it has the extra space.  Is weblate not being syncronised with GitHub changes?

**Comment by bilelmoussaoui** [comment]
@jayvdb Well, i contacted Weblate when i started this project in order to move from Transifex to it. But never got any answer and I totally lost hope on getting my project there. 
Well, i will implement few new stuff, fix all the issues and push a new version very soon! stay tuned :+1:

**Comment by jayvdb** [comment]
:-(

I believe only two of these fixes that was wrong.  The rest were good.

Anyway, I've seen lots of projects move to the hosted weblate recently, without any drama, and with lovely bidirectional sync.  I suggest emailing them again.

**Comment by bilelmoussaoui** [comment]
I have to enable webhooks and few stuff in order to make the auto update of translations files possible

**Comment by bilelmoussaoui** [comment]
Weblate is now pushing translations to the repo directly

**Comment by jayvdb** [comment]
@taufiqsumadi, so you can now redo your good translation fixes in weblate.

**Pull request merged**